src/librbd/migration/HttpClient.cc

```diff
@@ -460,7 +460,7 @@ class HttpClient<I>::HttpSession : public HttpSessionInterface {
     if (work->header_only()) {
       m_parser.emplace(std::move(*m_header_parser));
     }
-    response = std::move(m_parser->release());
+    response = m_parser->release();
```

Comment:
Just for my own education, what was the clang warning here? I was under the (mistaken?) impression that copy elision didn't function for assignment.

Reply:
Disregard -- you put it in the commit message.